### PR TITLE
[Doc] Update desc in Memory mgt

### PR DESCRIPTION
**Pull request by EsoragotoSpirit**
Signed-off-by: EsoragotoSpirit <wanglichen@starrocks.com>

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

File: docs/administration/Memory_management.md [hunk 1]
```diff
@@ -27,8 +27,8 @@ Explanation:
...
 | write_buffer_size | 104857600 |  The capacity limit of a single MemTable, exceeding which a disk swipe will be performed. |
-| load_process_max_memory_limit_bytes | 107374182400 | The total import memory limit `min(mem_limit * load_process_max_memory_limit_bytes, load_process_max_memory_limit_bytes)`. It is the actual available import memory threshold, reaching which a disk swipe will be triggered.  |
-| load_process_max_memory_limit_percent | 30 | The total import memory limit `min(mem_limit * load_process_max_memory_limit_percent, load_process_max_memory_limit_bytes)`. It is the actual available import memory threshold, reaching which the swipe will be triggered. |
+| load_process_max_memory_limit_bytes | 107374182400 | The maximum size limit of memory resources can be taken up by all load process on a BE node. Its value is the smaller one between `mem_limit * load_process_max_memory_limit_bytes` and `load_process_max_memory_limit_bytes`. It is the actual available import memory threshold, reaching which a disk swipe will be triggered.  |
```
Comment (evelynzhaojie) [inline docs/administration/Memory_management.md:30]: 1. The upper limit of memory resources that can be taken up by all load processes on a BE node. 2. the formula is not correct, different from the cn version

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by evelynzhaojie — APPROVED** [review]

**Review by hellolilyliuyi — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.5

**Pull request merged**